rest-api-spec/src/main/resources/rest-api-spec/api/security.saml_complete_logout.json

```diff
@@ -0,0 +1,27 @@
+{
+  "security.saml_complete_logout":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-saml-complete-logout.html",
+      "description":"Verifies the logout response sent from the SAML IdP"
+    },
+    "stability":"stable",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"]
```

Comment:
I'm not sure if this is used:
```
      "content_type": ["application/json"]
```

Reply:
Thanks for catching this. I think it should be included.